src/components/browserSupport/hooks/useCameraSupport.js

```diff
@@ -1,8 +1,12 @@
-import { iosSupportedWeb, isIOSWeb } from '../../../lib/utils/platform'
+import { iosSupportedWeb, isIOSWeb, isWebView } from '../../../lib/utils/platform'
 import useBrowserSupport from './useBrowserSupport'
 
 // if non-ios camera support is present on all browsers
 // for IOS camera is supported omnly in Safari or ios>=14.4 chrome+firefox
 export default (options = {}) => {
-  return useBrowserSupport({ ...options, checkOutdated: false, onCheck: () => !isIOSWeb || iosSupportedWeb })
+  return useBrowserSupport({
+    ...options,
+    checkOutdated: false,
+    onCheck: () => !isWebView && (!isIOSWeb || iosSupportedWeb),
```

Comment:
for QR code i think webview is fine, I dont think camerasupport should be blocked on webview

Reply:
https://github.com/GoodDollar/GoodDAPP/blob/f7fd34c41c345206b5c9c951abfcd7bb5f704f21/src/components/faceVerification/screens/IntroScreen.js#L147

If I remove it from useCameraSupport

```
  return useBrowserSupport({
    ...options,
    checkOutdated: false,
    onCheck: () => (!isIOSWeb || iosSupportedWeb)
```
isn't the local onCheck overriding the onCheck from ...options?

So then we should remove the local onCheck, or just re-arrange order?

Reply:
ok fixed